controller/helper/process-survey-instances.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**
+ * @module controller/helper/process-survey-instances
+ */
+
+/**
+ * Takes in a Survey Instances and processes it to get complence chart details
+ * @param {Surveys} list of survey instances
+ * @returns {Object}  Complience chart data
+ */
+function processSurveyInstances (surveys) {
+    return {
+      labels:pickupSubmissionTime(surveys),
+      datasets:calculateTimeLeft(surveys)
+    };
+}
+
+function pickupSubmissionTime(surveys) {
+  var dates = [];
+  dates.length = surveys.length;
+  for (var i = 0; i < surveys.length; i++) {
+    dates[i] = formatDate(surveys[i].startTime);
+  }
+  return dates;
+}
+
+function formatDate(date) {
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  minutes = minutes < 10 ? '0'+minutes : minutes;
+  var strTime = hours + ':' + minutes;
+  return date.getMonth()+1 + "/" + date.getDate() + "/" + date.getFullYear() + "  " + strTime;
```

Comment:
I would recommend using moment to handle formatting dates.
``` javascript
return moment(date).format('MM/DD/YYYY HH:mm');
```
would produce the format you want here.

Reference: http://momentjs.com/docs/#/displaying/format/